code/modules/events/brand_intelligence.dm

```diff
@@ -29,6 +39,13 @@
 
 /datum/event/brand_intelligence/tick()
 	if(!vendingMachines.len || !originMachine || originMachine.shut_up)	//if every machine is infected, or if the original vending machine is missing or has it's voice switch flipped
+		//VORESTATION Add - Effects when 'source' machine is destroyed/silenced
+		for(var/obj/machinery/vending/saved in infectedVendingMachines)

```

Comment:
To be honest, there wasn't anything else to add to the even, because  there's no penalty to not fixing the machines after 30 minutes. They just reset entirely. The machines aren't supposed to ever actually stop being infected, but I didn't want to ommit that without asking first.